config/datanode.example.toml

```diff
@@ -73,11 +73,9 @@ watch = false
 ## The runtime options.
 [runtime]
 ## The number of threads to execute the runtime for global read operations.
```

Comment:
Don't forget to update the configuration comments

Reply:
It's updated